mobile/src/main/res/values/strings.xml

```diff
@@ -54,9 +54,9 @@
     <string name="settings_clear_default_sitemap">Clear default sitemap</string>
     <string name="settings_no_default_sitemap">No default sitemap selected</string>
     <string name="settings_current_default_sitemap">Current default sitemap: %1$s</string>
-    <string name="settings_openhab_sslsettings">Encryption settings</string>
-    <string name="settings_openhab_alt_connection">Remote server settings</string>
-    <string name="settings_openhab_connection">Local server settings</string>
+    <string name="settings_openhab_sslsettings">Encryption</string>
+    <string name="settings_openhab_alt_connection">Remote server</string>
+    <string name="settings_openhab_connection">Local server</string>
```

Comment:
"... server" doesn't seem like the best fit. After all this is not primarily about the server but about the way the server is reached. How about something similar to "Local connection" or "Local address"... Not sure what's the best solution here.

Reply:
Or maybe just "local"?

Reply:
So, "connection" is already implied by the title of the prefernces category. Duplicating it would result in something similar as "Connection Settings", where we remove the "Settings", as it's already clear based on the context.

So, I think simply saying "Local", "Remote" is something which is short and still reasonable; and it would fit into the "Encryption" category name.

Reply:
Agreed 👍